media_softlet/linux/common/os/xe/mos_bufmgr_xe.c

```diff
@@ -1295,6 +1295,9 @@ mos_bo_alloc_xe(struct mos_bufmgr *bufmgr,
      */
     create.cpu_caching = alloc->ext.cpu_cacheable ? DRM_XE_GEM_CPU_CACHING_WB : DRM_XE_GEM_CPU_CACHING_WC;
 
+    if (strcmp(alloc->name, "MEDIA") == 0 || strcmp(alloc->name, "Media") == 0)
+        create.flags |= DRM_XE_GEM_CREATE_FLAG_SCANOUT;
```

Comment:
Cpu caching will become WC inside KMD when setting scanout, but umd/gmm may still record this surface with cpu cachable somewhere and want WB caching. Will have any conflict between umd/gmm and kmd?

Reply:
Added additional check to ensure that the caching is WC before we set the scanout flag.